scripts/shell/bt_nus_shell.py

```diff
@@ -91,14 +95,14 @@ def connect(self):
         if nrf_sd_ble_api_ver >= 3:
             if DFUAdapter.LOCAL_ATT_MTU > ATT_MTU_DEFAULT:
                 logger.info('BLE: Enabling longer ATT MTUs')
-                self.att_mtu = self.adapter.att_mtu_exchange(self.conn_handle)
+                self.att_mtu = self.adapter.att_mtu_exchange(self.conn_handle, BleSerial.LOCAL_ATT_MTU)
```

Comment:
How is this Python 3 related?

This PR seems to be mixing up three things:

1. cosmetics like class renames and whitespace changes
2. language changes required for converting to python 3
3. API changes in pc_ble_driver_py

I'm not sure if it's worth it for me to untangle this, so I will stop here and leave it up to others to merge this PR.